content/components/cards.mdx

```diff
@@ -20,40 +20,13 @@ icon: "rectangle"
 
 </RequestExample>
 
-### Icon Settings
-
-Adjust the icon type using the `iconType` property color by using the `color`
-property.
-
-<Card title="Colored Card" icon="lightbulb" iconType="duotone" color="#ca8b04">
-  Here is a card with a duotone yellow icon
-</Card>
-
-```jsx Card Icon Example
-<Card title="Colored Card" icon="lightbulb" iconType="duotone" color="#ca8b04">
-  Here is a card with a duotone yellow icon
-</Card>
-```
```

Comment:
wait why are we getting rid of this?

Reply:
Not really that useful, bloats the page a bit, and I think it works as well just having the prop makes it clear enough